src/Datadog.Trace/Logging/DatadogLogging.cs

```diff
@@ -141,6 +141,13 @@ private static string GetLogDirectory()
             //   - Path.GetTempPath
             if (logDirectory == null)
             {
+#if NETFRAMEWORK
+                var windowsDefaultDirectory = Path.Combine(Environment.GetFolderPath(Environment.SpecialFolder.CommonApplicationData), @"Datadog .NET Tracer", "logs");
+                if (Directory.Exists(windowsDefaultDirectory))
```

Comment:
I thought I commented on this in a recent PR (will try to find it), but we should try to create this folder if it doesn't exists, like we do for Linux below.

Reply:
I added this logic in 02d71db0c54b20a1f0be5eaafe7c497b51037732 . Let me know if this implementation is sufficient

Reply:
Oh, I didn't mean you should do it in this PR, but thanks.